SQL/0000-00-00-schema.sql

```diff
@@ -495,6 +495,7 @@ CREATE TABLE `files` (
   `Caveat` tinyint(1) default NULL,
   `TarchiveSource` int(11) default NULL,
   `ScannerID` int(10) unsigned default NULL,
+  `AcqOrderPerModality` int(11) default NULL,
```

Comment:
That field should probably be called: 'ModalityOrder' unless I misunderstood to match the code in LORIS-MRI (unless I misunderstood the code)